README.md

```diff
@@ -1,65 +1,256 @@
+The `kubectl port-forward` command temporarily forwards traffic
+from the local host to the MinIO Tenant. 
+
+- The `Web interface access` opens the MinIO Console interface for interacting
+  with and managing the MinIO Tenant.
+
+- The `Object storage access` provides both the classic MinIO web interface *and* general 
+  API access to the MinIO Object Store.
+
+Applications inside the Kubernetes cluster can access the MinIO Tenant through either the
+web interface service (`minio-tenant1-console`) *or* the object storage service
+(`minio-tenant1-hl`). 
+
+Applications outside of the Kubernetes cluster require the Kubernetes
+administrator to expose the service using Kubernetes resources like 
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) or a
+[Load Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). 
+
+# Expand a MinIO Tenant
+
+MinIO supports expanding an existing MinIO Tenant onto additional hosts and storage.
+
+- MinIO requires Kubernetes version 1.17.0 or later.
+
+- This procedure assumes the cluster contains a 
+  [namespace](https://github.com/minio/operator#create-a-namespace) for
+  the MinIO Tenant.
+
+- This procedure assumes the cluster contains a
+  [`StorageClass`](https://github.com/minio/operator#default-storage-class)
+  for the MinIO Tenant Persistent Volume Claims (`PVC`).
+
+The following `kubectl minio` command expands a MinIO Tenant with an additional
+4 nodes, 16 volumes, and added capacity of 16Ti. This configuration requires
+*at least* 16 [Persistent Volumes](https://github.com/minio/operator#Local-Persistent-Volumes).
 
 ```sh
-kubectl minio tenant create --name tenant1 --servers 4 --volumes 16 --capacity 16Ti -o > tenant.yaml
-kubectl apply -f tenant.yaml
+
+   kubectl minio tenant expand \
+      --name minio-tenant1 \
+      --servers 4 \
+      --volumes 16 \
+      --capacity 16Ti
+
 ```
 
-## Post Tenant Creation
+- The `--name` field specifies the name of the existing MinIO Tenant to expand.
 
-### Expanding a Tenant
+- The `--servers` field indicates the number of `minio` pods to deploy into the cluster.
+  The cluster *must* have at least one available Node per `minio` pod. 
 
-You can add capacity to the tenant using `kubectl minio` plugin, like this
+- The `--volumes` field indicates the total number of volumes in the Tenant. MinIO
+  generates a Persistent Volume Claim (`PVC`) for each volume and evenly distributes
+  volumes across each `minio` pod. The example above results in 4 volumes per `minio` pod.
 
+- The `--capacity` field indicates the total capacity of the cluster. MinIO determines the
+  amount of storage to request for each `pvc` by dividing the specified capacity by the
+  total number of volumes in the server. The example above results in 1Ti requested
+  capacity per volume.
+
+
+# Kubernetes Cluster Configuration
```

Comment:
Might be a good place to also stick in something around networking, like

## Networking Recommendations

MinIO can make efficient use of storage and can completely saturate network interfaces.

For example, consider a Node with locally-attached NVME SSD storage. The SSD can exhibit a maximum READ/WRITE performance of 2Gbps. If the node has 8 such drives, MinIO may drive a theoretical total of 16Gbps of traffic and overwhelm a 10GbE NIC. 

To avoid network bottlenecks, consider calculating the total possible READ/WRITE throughput of a single node and compare it against the total available network bandwidth provided by the node's NIC *and* the network infrastructure between that NIC and Clients. You can safely assume MinIO can drive up to 85% of theoretical drive throughput. 

Spitballing heavily here.